src/CalcViewModel/Common/Utils.cpp

```diff
@@ -86,11 +86,11 @@ bool Utils::IsLastCharacterTarget(_In_ wstring const &input, _In_ wchar_t target
     return !input.empty() && input.back() == target;
 }
 
-//return wstring after removing characters like space, comma, and double quotes
+//return wstring after removing characters like space, comma, double quotes, and monetary prefix currency symbols supported by the Windows keyboard
 wstring Utils::RemoveUnwantedCharsFromWstring(wstring input)
 {
-    wchar_t unWantedChars[] = { L' ', L',', L'"', 8234, 8235, 8236, 8237 };
-    return RemoveUnwantedCharsFromWstring(input, unWantedChars, 6);
+    wchar_t unWantedChars[] = { L' ', L',', L'"', 165, 164, 8373, 36, 8353, 8361, 8362, 8358, 8377, 163, 8364, 8234, 8235, 8236, 8237 };
+    return RemoveUnwantedCharsFromWstring(input, unWantedChars, 18);
```

Comment:
may as well compute the size automatically:
```
sizeof(unWantedChars) / sizeof(*unWantedChars)
```

Reply:
Addressed below.  We should prefer using the standard containers and the `size` member function.